modules/ximgproc/include/opencv2/ximgproc/run_length_morphology.hpp

```diff
@@ -0,0 +1,153 @@
+/*
+ *  By downloading, copying, installing or using the software you agree to this license.
+ *  If you do not agree to this license, do not download, install,
+ *  copy or use the software.
+ *
+ *
+ *  License Agreement
+ *  For Open Source Computer Vision Library
+ *  (3 - clause BSD License)
```

Comment:
Please use short 3-line OpenCV license header (similar to perf_run_length_morphology.cpp file):
```
// This file is part of OpenCV project.
// It is subject to the license terms in the LICENSE file found in the top-level directory
// of this distribution and at http://opencv.org/license.html.
```